### DISCUSSION TITLE
Where to ask questions about individual packages?

### DISCUSSION THREAD

**Discussion by cyberdynesoftware** [Homebrew/discussions#471] [Everyday usage]
To my understanding both the Homebrew discussions as well as github issues for Homebrew deal with brew itself, not with individual packages. I couldn't find a place to ask questions regarding individual packages.

My questions regards the webp package and more specifically, why the gif2webp executible is missing.

**Comment by SMillerDev** [comment]
Discussions is a good place for questions about anything brew related. But generally the answer for "why is x not enabled" is, because it's not enabled by default and nobody made a pull request for it yet.

**Comment by hyuraku** [comment]
> My questions regards the webp package and more specifically, why the gif2webp executible is missing.

Because `webp` install configuration includes [`"--disable-gif"`](https://github.com/Homebrew/homebrew-core/blob/07c76f5b6cbc5afc98ff510165f83c828614cd31/Formula/webp.rb#L32), `gif2webp` isn't  in`/usr/local/Cellar/webp/1.1.0/bin`.

**Reply by rdp** [reply]
My guess is typically something to enable it would be accepted.  Unless a package gets too complicated.  I thought there was a time when they started ripping out all the dependencies from FFmpeg but appears they've crept back in so...probably they're OK with adding new options these days :)

**Comment by rdp** [comment]
I guess you could look at the git history and also email historical contributors...

**Reply by SMillerDev** [reply]
Please don't do this, many contributors only contribute one fix and I doubt they would be open for support questions.

**Comment by rdp** [comment]
Major contributors?  Or I guess GitHub discussions then.  Or a discussion
with the project that your are intending to build, their maintainers :)

On Tuesday, January 12, 2021, Sean Molenaar <notifications@github.com>
wrote:
> Please don't do this, many contributors only contribute one fix and I
doubt they would be open for support questions.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci5.googleusercontent.com/proxy/l_x_ZosuAOPtwxc7D9Mlb_Y_F4OADKvPn-RCydEb9JYGGooFFFS2ETCsGYUceyGDdVjKH5sddL_b2T-wC6ys7ZgP1yiVOHCtS5ES0oFoxgVbIHqUHGjVj5-WNPnkTTRD3QbP95kr9G_4BLY23gyMxSFIiaDL7Kyh-nSNtVuYcEsERGV2naJHifbICWC-eFto9b6xtAu8mPxJyo4s00npnDFoCxMSyG60LFIsdRLhDIN2BEn8TX8y=s0-d-e1-ft#https://github.com/notifications/beacon/AAADBUGR77XQIDFLOBHCXJTSZP3RVA5CNFSM4VZMAMKKYY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAACDMYA.gif>
